scripts/migration/statefulset/crdbcluster-template.json

```diff
@@ -0,0 +1,96 @@
+{
+  "apiVersion": "crdb.cockroachlabs.com/v1alpha1",
+  "kind": "CrdbCluster",
+  "metadata": {
+    "name": env(release_name) + "-cockroachdb",
+    "namespace": env(namespace)
+  },
+  "spec": {
+    "dataStore": {},
+    "features": [
+      "reconcile",
+      "reconcile-beta"
+    ],
+    "mode": "MutableOnly",
+    "regions": [
+      {
+        "cloudProvider": env(cloud_provider),
+        "code": env(region),
```

Comment:
cloud_provider and region has to be provided by the user. Right now it is not mentioned anywhere